### PR TITLE
Add a switch to select appearance manually

### LINKED ISSUE #58: [BUG] Dark mode lacks contrast
#### TODO
* Color scheme in dark mode should be updated to adjust the brightness.
* Whether to add a manual switch on light and dark mode depends on further information.

Previous issue: #47. Further updates will go into this thread.

Possibly related to user: @louie-github. Sorry for my bothering and you are free to mute this thread.

### PR DESCRIPTION
**Pull request by YunzheZJU**
This PR implements and resolves #58, where the switch for appearance is requested.

#### TODO
  - [x] Japanese translation

### CODE REVIEW

File: src/popup/i18n/locales/ja.json5 [hunk 1]
```diff
@@ -43,6 +43,22 @@
...
+        // Description for appearance
+        description: "TODO:Change the appearance to your preferred theme. Changes are only applied to the current browser",
+        // Supported appearances
+        appearances: {
+          // The theme follows the current OS setting
+          device: "TODO:Follow device",
+          // Default light theme
+          light: "TODO:Default light",
+          // A theme which comes with a contrast not so strong as black-on-white and which best fits to night use
+          dark: "TODO:Dimmed dark"
```
Comment (YunzheZJU) [inline src/popup/i18n/locales/ja.json5:59]: This pull-request needs translations in Japanese for these strings marked with `TODO:`.
Please make a comment, propose a suggestion here or make another pull-request to this branch `appearance` if you are interested.

Comment (taizan-hokuto) [reply src/popup/i18n/locales/ja.json5:59]:
```
       // Label for appearance
        label: "外観",
        // Description for appearance
        description: "外観を好きなテーマに変更します。変更は現在のブラウザにのみ適用されます。",
        // Supported appearances
        appearances: {
          // The theme follows the current OS setting
          device: "端末の設定に従う。",
          // Default light theme
          light: "ライト（既定）",
          // A theme which comes with a contrast not so strong as black-on-white and which best fits to night use
          dark: "ダーク"
```

Referenced by Firefox settings.

File: src/popup/i18n/locales/ja.json5 [hunk 2]
```diff
@@ -43,6 +43,22 @@
           "zh-CN": "简体中文"
         }
       },
+      // Prompt users to switch the appearance
+      appearance: {
+        // Label for appearance
+        label: "TODO:Appearance",
```
Comment (YunzheZJU) [inline src/popup/i18n/locales/ja.json5:49]:
```suggestion
        label: "デザイン",
```

File: src/popup/i18n/locales/ja.json5 [hunk 3]
```diff
@@ -43,6 +43,22 @@
...
+        // Description for appearance
+        description: "TODO:Change the appearance to your preferred theme. Changes are only applied to the current browser",
```
Comment (YunzheZJU) [inline src/popup/i18n/locales/ja.json5:51]:
```suggestion
        description: "外観を好きなテーマに変更します。変更は現在のブラウザにのみ適用されます",
```

File: src/popup/i18n/locales/ja.json5 [hunk 4]
```diff
@@ -43,6 +43,22 @@
...
+        // Supported appearances
+        appearances: {
+          // The theme follows the current OS setting. This is the default selection
+          device: "TODO:Follow device",
```
Comment (YunzheZJU) [inline src/popup/i18n/locales/ja.json5:55]:
```suggestion
          device: "端末の設定に従う",
```

File: src/popup/i18n/locales/ja.json5 [hunk 5]
```diff
@@ -43,6 +43,22 @@
...
+          // Default light theme which is designed for daily use
+          light: "TODO:Default light",
```
Comment (YunzheZJU) [inline src/popup/i18n/locales/ja.json5:57]:
```suggestion
          light: "ライトテーマ",
```

File: src/popup/i18n/locales/ja.json5 [hunk 6]
```diff
@@ -43,6 +43,22 @@
...
+          // A theme which comes with a contrast not so strong as black-on-white and which best fits to night use
+          dark: "TODO:Dimmed dark"
```
Comment (YunzheZJU) [inline src/popup/i18n/locales/ja.json5:59]:
```suggestion
          dark: "ダークテーマ"
```

**Review by YunzheZJU — COMMENTED** [review]
Use translations provided by taizan sama.
I modified some of them, following the translation found in Youtube's settings

File: src/popup/i18n/locales/ja.json5 (hunk 1 above)
Comment (YunzheZJU) [reply src/popup/i18n/locales/ja.json5]: Thank you!! I will adopt it!

**Pull request merged**